Gopkg.lock

```diff
@@ -1214,6 +1215,14 @@
   revision = "d2d2541c53f18d2a059457998ce2876cc8e67cbf"
   version = "v0.9.1"
 
+[[projects]]
+  digest = "1:c4b5592c342f273e18de68e16957f536f6648f2e90c9d7ece653ac90c22079a9"
+  name = "gopkg.in/ini.v1"
```

Comment:
New dependency. Can we trace back why it was added? Something like `go mod why`?

Reply:
It's used by viper.
```
$ gr gopkg.in/ini.v1 vendor
vendor/gopkg.in/ini.v1/README.md:27:$ go get gopkg.in/ini.v1
vendor/gopkg.in/ini.v1/README.md:35:- [API Documentation](https://gowalker.org/gopkg.in/ini.v1)
vendor/gopkg.in/ini.v1/.travis.yml:18:  - ln -s $HOME/gopath/src/github.com/go-ini/ini $HOME/gopath/src/gopkg.in/ini.v1
vendor/gopkg.in/ini.v1/.travis.yml:19:  - cd $HOME/gopath/src/gopkg.in/ini.v1
vendor/github.com/spf13/viper/go.mod:46:	gopkg.in/ini.v1 v1.51.0
vendor/github.com/spf13/viper/go.sum:185:gopkg.in/ini.v1 v1.51.0 h1:AQvPpx3LzTDM0AjnIRlVFwFFGC+npRopjZxLJj6gdno=
vendor/github.com/spf13/viper/go.sum:186:gopkg.in/ini.v1 v1.51.0/go.mod h1:pNLf8WUiyNEtQjuu5G5vTm06TEv9tsIgeAvK8hOrP4k=
vendor/github.com/spf13/viper/viper.go:48:	"gopkg.in/ini.v1"
```